lib/tasks/index.rake

```diff
@@ -8,7 +8,7 @@ def index_doc(filter_tags, doc_list, get_content)
   rebuild = ENV['REBUILD_DOC']
   rerun = ENV['RERUN'] || rebuild || false
   
-  filter_tags.call(rebuild).sort_by { |tag| Version.version_to_num(tag.first[1..-1])}.each do |tag|
+  filter_tags.call(rebuild).sort_by { |tag| Version.version_to_num(tag.first[1..-1])}.reverse.each do |tag|
```

Comment:
I can easily revert this one, and I understand what we trying to prevent here, but on the other hand it feels hacky to sort by oldest version first because we are afraid the insertions can mess older/already existing versions no? shouldn't the correct approach be to have a test ensuring that doesn't happen?

Reply:
I agree that in an ideal world there wouldn't be mysterious and hacky ordering constraints. And maybe there aren't even, and we're just being paranoid. But without a strong reason to change the order, I'd just as soon leave it as-is (especially because I think your use case of "start recent and then abort" is better served by just asking the task to build one specific version via `REBUILD_DOC`).